railties/lib/rails/commands/server/server_command.rb

```diff
@@ -7,51 +7,14 @@
 module Rails
   class Server < ::Rack::Server
     class Options
-      DEFAULT_PID_PATH = File.expand_path("tmp/pids/server.pid").freeze
-
       def parse!(args)
-        args, options = args.dup, {}
-
-        option_parser(options).parse! args
-
-        options[:log_stdout] = options[:daemonize].blank? && (options[:environment] || Rails.env) == "development"
-        options[:server]     = args.shift
-        options
-      end
-
-      def option_parser(options) # :nodoc:
-        OptionParser.new do |opts|
-          opts.banner = "Usage: rails server [puma, thin etc] [options]"
```

Comment:
If `argument` used, need to specify the "server" argument twice to specify server(e.g. `./bin/rails server server webrick`).  
Therefore, override `.banner` instead.